### PR TITLE
Implement List All Orders API Endpoint

### LINKED ISSUE #2: List all orders
As a system administrator
I need an API endpoint that allows me to list all orders in the system
So that I can oversee and manage the order processing and fulfillment process more effectively.

Details and Assumptions
Each order's details in the list should include key information such as order ID, customer ID, order date, total amount, and status.

 ### Acceptance Criteria  
   
 ```gherkin
When a GET request is issued to /orders
Then the system should return a list of all orders with their key details
```

### PR DESCRIPTION
**Pull request by TheAriaki**
Implemented List All Orders API Endpoint using the template from [https://github.com/nyu-devops/sample-accounts/tree/master](url) to Fixes #2 

The following aspects have been implement/addressed

Model Serialization Update: Updated the serialize method in order.py to ensure the status field is correctly serialized, accommodating both string and enum representations.

API Endpoint Implementation: Implemented the /orders GET endpoint in routes.py, facilitating the retrieval of all orders. This endpoint utilizes the Order.all() method to fetch all orders from the database and serializes them for the response.

Test Coverage: Added a new test case in test_routes.py to validate the functionality of the /orders endpoint. The test ensures that the endpoint correctly returns a list of orders with their respective details, matching the data in the database.

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -20,12 +20,12 @@
 This service implements a REST API that allows you to manage Orders for a financial service.
 """
 
-from flask import jsonify
+from flask import jsonify, request
```
Comment (IhwaHussain) [inline service/routes.py:23]: Change to ```from flask import jsonify``` to comply with pylint standards, as that code is imported but not used.

File: service/routes.py [hunk 2]
```diff
@@ -20,12 +20,12 @@
...
 
 # , request, url_for, abort
 from flask import current_app as app  # Import Flask application
 
-# from service.models import Order, Item
+from service.models import Order, Item
```
Comment (IhwaHussain) [inline service/routes.py:28]: Change to ```from service.models import Order``` to comply with Pylint standards, as the `Item` code is imported but not used

**Review by IhwaHussain — CHANGES_REQUESTED** [review]
Code looks good and adheres to RESTful API guidelines while listing the orders in the database. There are some PyLint errors that should be resolved, and then we're good to go.

File: service/routes.py (hunk 2 above)
Comment (TheAriaki) [reply service/routes.py]: Fixed the Pylint errors

**Pull request merged**